src/app/runtime/wac/components/install.component.ts

```diff
@@ -90,22 +97,56 @@ export class InstallComponent {
         this._targetHost = this.appContext.activeConnection.nodeName
     }
 
-    public verifyLocation(value: string, allowEmpty: boolean) {
-        this.invalid = value ? !urlValidationRegex.test(value) && !windowsPathValidationRegex.test(value) : !allowEmpty
+    clearWarnings(event: Event) {
+        (<HTMLElement> event.target).classList.remove('background-warning')
     }
 
-    public install() {
-        this._inProgress = true
-        let args: any = {
-            command: 'install',
-            adminAPILocation: this._adminAPILocation,
+    private verifyLocation(fieldName: string, location: string, allowEmpty: boolean): string {
+        if (location) {
+            if (!urlValidationRegex.test(location) && !windowsPathValidationRegex.test(location)) {
+                return `Invalid ${fieldName}: ${location}`
+            }
+        } else {
+            if (!allowEmpty) {
+                return `${fieldName} cannot be empty`
+            }
         }
-        if (this._dotnetCoreLocation) {
-            args.dotnetCoreLocation = this._dotnetCoreLocation
+
+        // NOTE: This code is injected here to prevent user using shared drive because it would be considered double hop with WinRM and wouldn't work
+        // This needs to be revisited when Windows Admin Center completed their work on CredSSP
+        if (location && location.startsWith(`\\\\`)) {
+            return `Currently installation from shared drive is not supported, the issue is being tracked on https://github.com/Microsoft/IIS.WebManager/issues/239`
+        }
+    }
+
+    private verifyLocationPrompt(prompt: ElementRef, allowEmpty: boolean): boolean {
+        let fieldName = prompt.nativeElement.querySelector('label').textContent
+        let textBox = prompt.nativeElement.querySelector('input')
+        textBox.classList.remove('background-warning')
+        this.userInputError = this.verifyLocation(fieldName, textBox.value, allowEmpty)
+        let valid = this.userInputError == null
+        if (!valid) {
+            textBox.classList.add('background-warning')
+            textBox.focus()
+        }
+        return valid
+    }
+
+    public install() {
+        if (this.verifyLocationPrompt(this.apiPrompt, false)
+            && this.verifyLocationPrompt(this.dotnetPrompt, true)) {
+            this._inProgress = true
+            let args: any = {
+                command: 'install',
+                adminAPILocation: this._adminAPILocation,
+            }
+            if (this._dotnetCoreLocation) {
+                args.dotnetCoreLocation = this._dotnetCoreLocation
+            }
+            var sub = this.runtime.PrepareIISHost(args).subscribe(_ => {}, _ => {}, () => {

```

Comment:
Please add error handling when the provided Url returns non 200 status value. 
In that case, I think we should inform users the provided URL is not valid so that users can retry.

Reply:
Good idea, fixed